docs/eventing/experimental-features.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Experimental features"
+weight: 99
+type: "docs"
+---
+
+# Experimental features
+
+In Knative Eventing we want to keep the innovation alive, experimenting and delivering new features without affecting the stability of the project.
+
+In order to achieve that goal, we have a process to include new features.
+This allows users like you to try out new features and provide feedback back to the project.
```

Comment:
Super nit, and this may just be how I read this, but I think specifying that "Knative Eventing" wants to keep the innovation alive reads a bit like we're saying that Serving doesnt 😅 which is obviously not the case! WDYT about something like: 
```suggestion
In Knative we want to keep the innovation alive, experimenting and delivering new features without affecting the stability of the project.

In order to achieve that goal in Knative Eventing, we have a process to include new features.
This allows users like you to try out new features and provide feedback back to the project.
```